hypha/apply/dashboard/views.py

```diff
@@ -163,6 +163,7 @@ def get(self, request, *args, **kwargs):
             'active_statuses_filter': active_statuses_filter,
             'my_submissions': my_submissions,
             'my_inactive_submissions': my_inactive_submissions,
+            'my_flagged': self.get_my_flagged(self.request, qs),
```

Comment:
Is "qs" the correct collection off submissions to use here?

Reply:
Yeah, but it could have been named better. The admin dashboard view uses [submissions](https://github.com/OpenTechFund/hypha/blob/d7181b723de077d311477b58763f7b5d4bdfa8ad/hypha/apply/dashboard/views.py#L30). Also, this context code should be in [get_context_data](https://github.com/OpenTechFund/hypha/blob/d7181b723de077d311477b58763f7b5d4bdfa8ad/hypha/apply/dashboard/views.py#L210) method instead of get. We will refactor it in dashboard view code refactoring.